src/frontends/onnx/frontend/src/op/com.microsoft/pad.hpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/deprecated.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
+
+#include "onnx_import/core/node.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector pad(const Node& node);
+}  // namespace set_1
+}  // namespace op
+}  // namespace onnx_import
+}  // namespace ngraph
```

Comment:
could you please add a test using a test model like there https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/tests/models ?

Reply:
Actually I am not sure after I have added the test model how can I run the tests locally?

Reply:
Test models are automatically converted to .onnx files during build of OpenVINO tests (make sure that appropriate dependencies are installed; cmake will notify about it if something is missed, just read cmake output).

For example, we have a model file https://github.com/openvinotoolkit/openvino/blob/749ed9dec7f2817fb3f27838ea3d307bf14c9ebf/src/core/tests/models/ir/split_equal_parts_2d.prototxt#L15, which is converted to `onnx/split_equal_parts_2d.onnx` and used in this test:
https://github.com/openvinotoolkit/openvino/blob/749ed9dec7f2817fb3f27838ea3d307bf14c9ebf/src/frontends/onnx/tests/onnx_import_reshape.in.cpp#L487-L498
To run the tests, go to `<source dir>/bin/intel64/Releases` and run `ov_onnx_frontend_tests` binary

Reply:
Do I need to buld OpenVino before running the tests?

Reply:
FYI: I have just forked cloned the repo and added the code. I'm unsure if the code changes are correct or wrong as I was not able to test it.

Reply:
sure, OpenVINO tests are built together with OpenVINO if you enable `-DENABLE_TESTS` in cmake

Reply:
Should I follow these steps https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build_windows.md? I'm setting it up on a windows machine

If I follow these steps the build creates new files, which identifies as untracked files by git, what should I do in this case?

Reply:
Error while running `cmake --build . --config Release --verbose -j8` with flag `-DENABLE_TESTS=ON`.

Error trace:
Unknown argument -DENABLE_TESTS=ON
Usage: cmake --build <dir>             [options] [-- [native-options]]
       cmake --build --preset <preset> [options] [-- [native-options]]
       
 Could you help me with this @p-wysocki @gkrivor @ilya-lavrenov. Thanks.

Reply:
Please, follow build guides first https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md

Reply:
@kshitij01042002 Here's a CMake command I'm usually using, it should work fine:

```bash
cmake .. -DENABLE_TESTS=ON -DNGRAPH_UNIT_TEST_ENABLE=ON -DENABLE_FUNCTIONAL_TESTS=ON -DENABLE_OPENCV=ON -DENABLE_INTEL_CPU=ON -DENABLE_INTEL_GNA=OFF -DENABLE_INTEL_GPU=OFF -DENABLE_PYTHON=ON -DENABLE_NCC_STYLE=OFF -DCMAKE_INSTALL_PREFIX=/home/pwysocki/openvino_dist -DENABLE_DEBUG_CAPS=ON -DENABLE_CPU_DEBUG_CAPS=ON -DPYTHON_EXECUTABLE=`which python`
```
Please let mi know if it works for you. After that I usually just ran `make -jX install`.